build-system/tasks/runtime-test.js

```diff
@@ -28,22 +28,37 @@ var webserver = require('gulp-webserver');
 var app = require('../test-server').app;
 var karmaDefault = require('./karma.conf');
 
+
+/**
+ * Logs a message if tests are being run during local development.
+ */
+function logLocal() {
+  if (!process.env.TRAVIS) {
+    util.log(util.colors.green.apply(null, arguments));
+  }
+}
+
 /**
  * Read in and process the configuration settings for karma
  * @return {!Object} Karma configuration
  */
 function getConfig() {
   if (argv.safari) {
+    logLocal(util.colors.yellow('--safari:'), 'Running tests on Safari.');
```

Comment:
Re: `getConfig`, the only place where it is called is when tests are being run, hence the message.

Re: the map, good idea. Coming up.